src/Spryker/Zed/EventBehavior/Business/Model/EventEntityTransferFilter.php

```diff
@@ -145,6 +149,104 @@ public function getEventTransfersAdditionalValues(array $eventTransfers, string
         return array_unique($additionalValues);
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\EventEntityRequestTransfer $eventTransferRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\EventEntityResponseTransfer
+     */
+    public function getEventTransferValuesWithTimestamps(EventEntityRequestTransfer $eventTransferRequestTransfer): EventEntityResponseTransfer
```

Comment:
Not quite sure about the name. I think these methods are filters for the event transfers. How about name it `filterEventTransferData`, `hydrateEventTransferData`, `collectEventTransferData`.

I would not add the timestamp detail into the naming; that makes it harder to re-use.